sweepai/handlers/on_comment.py

```diff
@@ -100,7 +100,7 @@ def on_comment(
     if pr is None:
         pr = repo.get_pull(pr_number)
     pr_title = pr.title
-    pr_body = pr.body or ""
+    [ORIGINAL_CODE]
```

Comment:
Added a check to see if the phrase "🎉 Latest improvements to Sweep:" is present in the PR description.
If the phrase is found, everything after this phrase is removed from the PR description.
If the phrase is not found, no changes are made to the PR description.